data/json/monsters/drones.json

```diff
@@ -185,5 +185,20 @@
       "PRIORITIZE_TARGETS",
       "HIT_AND_RUN"
     ]
+  },
+  {
+    "id": "mon_tazer_hack",
+    "copy-from": "base_drone",
+    "type": "MONSTER",
+    "name": { "str": "tazer hack" },
+    "description": "An automated drone, this small quadcopter robot appears to have a tazer strapped to the front. While extrememly fast, it is very fragile.",
+    "hp": 5,
+    "diff": 10,
+    "speed": 300,
+    "color": "cyan",
+    "armor_cut": 2,
+    "armor_bullet": 1,
+    "revert_to_itype": "bot_tazer_hack",
+    "special_attacks": [ [ "ZAPBACK" ], [ "TAZER", 5 ] ]
```

Comment:
Replace `ZAPBACK` special attack with `"special_when_hit": [ "ZAPBACK", 75 ]` line. Enter any value you find appropriate.